spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Linear.scala

```diff
@@ -279,8 +280,6 @@ class Linear(
   }
 
   override def zeroGradParameters(): Unit = {
```

Comment:
maybe we can just remove the method. I remember the default impl is do nothing, right?